NOTICE

```diff
@@ -141,5 +141,10 @@ This software includes source code and libraries developed by:
     Copyright 2008, http://www.snakeyaml.org
     Licensed under Apache License, Version 2.0
 
+    Spack
+    https://github.com/spack/spack
+    Copyright 2013-2023 Lawrence Livermore National Security, LLC and other Spack Project Developers. See the top-level COPYRIGHT file for details.
+    Licensed under Apache License, Version 2.0
```

Comment:
Not sure this should be added. What code is taken from Spack?

Reply:
So, I added this because I used their Dockerfile template as a starting point for ours:
https://github.com/spack/spack/blob/develop/share/spack/templates/container/Dockerfile

Not sure whether this is the best way to acknowledge it within Nextflow, what do you reckon?

Reply:
Then, the apache header should go also in the dockerfile

Reply:
done!